R/fb_player_scouting.R

```diff
@@ -21,25 +22,27 @@
 #'
 #' # to filter for the last 365 days:
 #' fb_player_scouting_report(player_url = "https://fbref.com/en/players/d70ce98e/Lionel-Messi",
-#' pos_versus = "primary") %>% dplyr::filter(scouting_period == "Last 365 Days")
+#' pos_versus = "primary", league_comp_name = "Last 365 Days Men's Big 5 Leagues, UCL, UEL")
 #'
 #' # to get secondary positions
 #' fb_player_scouting_report(player_url = "https://fbref.com/en/players/d70ce98e/Lionel-Messi",
 #' pos_versus = "secondary")
 #'
-#' # for the 2020-2021 La Liga season
+#' # for the last 365 days and also the 2022 WC
 #' fb_player_scouting_report(player_url = "https://fbref.com/en/players/d70ce98e/Lionel-Messi",
-#' pos_versus = "secondary") %>% dplyr::filter(scouting_period == "2020-2021 La Liga")
+#' pos_versus = "secondary",
+#' league_comp_name = c("Last 365 Days Men's Big 5 Leagues, UCL, UEL", "2022 World Cup"))
 #' })
 #' }
-fb_player_scouting_report <- function(player_url, pos_versus, time_pause=3) {
+fb_player_scouting_report <- function(player_url, pos_versus, league_comp_name=NULL, time_pause=3) {
```

Comment:
Defaulting this to NULL. Should users want to get reports for specific leagues/comps, they can. This can be either one, or multiple values